CRM/Core/Session.php

```diff
@@ -550,12 +550,9 @@ public static function storeSessionObjects($reset = TRUE) {
    * @return int|null
    *   contact ID of logged in user
    */
-  public static function getLoggedInContactID() {
-    $session = CRM_Core_Session::singleton();
-    if (!is_numeric($session->get('userID'))) {
-      return NULL;
-    }
-    return (int) $session->get('userID');
+  public static function getLoggedInContactID(): ?int {
+    $userId = CRM_Core_Session::singleton()->get('userID');
```

Comment:
Hmm - this userId vs userID contiues to be a case where you are adding the freud version & I'm removing it in favour of the ID version....

Reply:
@eileenmcnaughton The battle for `fooBarID` was lost before it was begun. No matter how many times one adds `fooBarID` to `civicrm-core.git`, the `fooBarId` notation will still be more common in the broader world of Drupal, Joomla, WordPress, Symfony, DOM, Java, ad nauseum.

I try to imagine why this particular convention bothers you so much. Maybe it's like the Nose Problem. When I was 14, I realized: "I never see noses." They exist, so you should see them, right? But I never did. So then I started looking. Mary had a nose; Andy had a nose; Ruth had nose; Ben too! They were everywhere! _Everywhere!_ The noses wouldn't go away! Every single person had one of those monstrous lumps sticking out of their face! Aaah! The horror! How could they live like that? How could I? Eventually, I held my eyes shut for 30 solid seconds -- and it reset my brain. Then I resolved never to do that again. While it's true that every face has a nose, it's not worth focusing on them.

Reply:
the reason it bothers me is that there was discussion a long time ago about what our standard should be - I felt there was an understanding & have been implementing it ever since but then realised that we were still actively implementing 2 different variants. If we had opted to camel id rather than ID when we discussed it way back when I would have been trying to make everything id not ID